tests/test_stl_binders.cpp

```diff
@@ -173,6 +237,11 @@ TEST_SUBMODULE(stl_binders, m) {
     py::bind_map<MutuallyRecursiveContainerPairMV>(m, "MutuallyRecursiveContainerPairMV");
     py::bind_vector<MutuallyRecursiveContainerPairVM>(m, "MutuallyRecursiveContainerPairVM");
 
+    // Bind with private inheritance + `using` directives.
+    // Feel free to add new `using` directives there.
```

Comment:
Maybe delete this comment line?

Or did you mean to suggest something is not covered even though it could be? What?

Reply:
Currently I'm thinking (but not sure, please correct):

It might be best to delete the comment here and instead have something like this in `UserVectorLike` and/or `UserMapLike`:

```
// This is only a subset of the member functions, as needed at the time.
```

I think that's really all that's needed.